src/modules/Builder/features/ActivityItems/LeftBar/LeftBar.tsx

```diff
@@ -1,33 +1,31 @@
-import { useRef } from 'react';
 import { useTranslation } from 'react-i18next';
-import { Button } from '@mui/material';
 
+import { StyledBuilderBtn, StyledHeadlineLarge } from 'shared/styles/styledComponents';
```

Comment:
this row can be merged with row 5: 
`import { StyledBuilderBtn, StyledHeadlineLarge, theme } from 'shared/styles`